src/modules/LanUdevAuto.rb

```diff
@@ -205,7 +207,7 @@ def Write
       end
 
       if !rules.empty? && AllowUdevModify()
-        SetAllLinksDown()
+        SetAllLinksDown() if !Linuxrc.usessh
```

Comment:
I *guess* it is also affected but I would not mention it if we haven't *tested* it.